daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -127,3 +127,15 @@ message NodeRollback { // *since version 14*
 }
 
 // architecture-handbook-entry-end: Nodes
+
+message FatContractInstance {
+  bytes contract_id = 1;
+  com.daml.lf.value.Identifier template_id = 2;
+  bytes create_arg = 3;
+  bytes contract_key = 4;
+  repeated string maintainers = 5;
+  repeated string non_maintainer_signatories = 6;
+  repeated string non_signatory_stakeholders = 7;
+  int64 createTime = 8;
```

Comment:
on the ledger API this is called `created_at`. Any reason to use a different name here?

Reply:
:+1:  I second that

Reply:
changed.